ampd/src/asyncutil/future.rs

```diff
@@ -61,12 +70,10 @@ mod tests {
 
     #[tokio::test]
```

Comment:
make this `start_paused` so we don't actually have to wait seconds for the test to pass